src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -20,29 +19,20 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        internal int _version;
         private List<Cookie> _list = new List<Cookie>();
 
         private DateTime _timeStamp = DateTime.MinValue;
         private bool _hasOtherVersions;
 
-        private readonly bool _isReadOnly;
-
         public CookieCollection()
         {
-            _isReadOnly = true;
-        }
-
-        internal CookieCollection(bool IsReadOnly)
```

Comment:
@justinvp How do know that it is unused code?  Simply scanning the CURRENT source code in GitHub is not sufficient to be able to draw that conclusion.  We have not ported all the code to GitHub yet.